test/fake-xhr.js

```diff
@@ -0,0 +1,92 @@
+/** @flow */
```

Comment:
similar naming thing; is a file just exports a class, maybe the file should be named like the class e.g. FakeXHR (similar to how we export components in CycleDash)?